index.php

```diff
@@ -0,0 +1,13 @@
+<?PHP
+
+echo "Hi there, here is some code!";
+
+$apples = 12;
+$oranges = 23;
+
+$totalFruit = $apples + $oranges + $grapes;
```

Comment:
$grapes is not defined anywhere, so this calculation is incorrect.

Reply:
Thanks! I added some grapes so this should be fixed now.